CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ Please follow [the Keep a Changelog standard](https://keepachangelog.com/en/1.0.
 
 ## [Unreleased]
 
+## [3.4.3]
+
+### Fixed
+
+* The documentation regarding the topic "Dealing with endpoint duplicates."
```

Comment:
No need to bump the version and add a change to changelog -- this change does not affect the users

Reply:
Removed it from here